### ISSUE TITLE
Are there papers on lane detection using ERFNet?

### ISSUE THREAD

**Issue by xxzcxb1314** [cardwing/Codes-for-Lane-Detection#165]
could you send me the paper of ERFNet about lane detection?

**Comment by cardwing** [comment]
ERFNet is originally designed for semantic segmentation and has not been applied in lane detection. I just release a lane detection code that uses ERFNet as the backbone and achieves very good performance. Applying SAD to ERFNet should give you much better performance.

**Comment by zimurui** [comment]
where is the SAD code? I failed to find it in this github..

**Comment by cardwing** [comment]
It is put in ENet-Label-Torch (model definition and loss function).

**Comment by xxzcxb1314** [comment]
Thank you for your reply.I will have a try.



---Original---

**Comment by xxzcxb1314** [comment]
Hi,when I train the model, I does not get a good acc,could you give me some suggestions?
My training results are shown in the attached picture.

------------------ 原始邮件 ------------------
发件人: "MaybeShewill-CV"<notifications@github.com>;
发送时间: 2019年10月8日(星期二) 晚上8:39
收件人: "cardwing/Codes-for-Lane-Detection"<Codes-for-Lane-Detection@noreply.github.com>;
抄送: "2627084603"<2627084603@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [cardwing/Codes-for-Lane-Detection] Are there papers on lanedetection using ERFNet? (#165)




It is put in ENet-Label-Torch (model definition and loss function).

**Comment by cardwing** [comment]
Using the provided training script should give you similar results. I have just tested the training script and it runs well.

**Comment by xxzcpsmemeda** [comment]
> where is the SAD code? I failed to find it in this github..



> It is put in ENet-Label-Torch (model definition and loss function).



> Using the provided training script should give you similar results. I have just tested the training script and it runs well.

**Comment by xxzcpsmemeda** [comment]
Hi,I want to know if there is no line in the photos，how to fit the lane?



---Original---

**Comment by xxzcpsmemeda** [comment]
---原始邮件---
发件人: "2627084603"<2627084603@qq.com&gt;
发送时间: 2019年11月29日(星期五) 上午10:03
收件人: "MaybeShewill-CV"<reply@reply.github.com&gt;;
主题: 回复： [cardwing/Codes-for-Lane-Detection] how to compute the run time? (#188)


Could you tell me run time is the average time taken to test 100 images or a batch of images??




------------------ 原始邮件 ------------------
发件人:&nbsp;"MaybeShewill-CV"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月29日(星期五) 上午8:30
收件人:&nbsp;"cardwing/Codes-for-Lane-Detection"<Codes-for-Lane-Detection@noreply.github.com&gt;;
抄送:&nbsp;"2627084603"<2627084603@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [cardwing/Codes-for-Lane-Detection] how to compute the run time? (#188)




In the testing phase.
